### ISSUE TITLE
is there any way to update cell without re-render whole grid

### ISSUE THREAD

**Issue by bayaroch-um** [silevis/reactgrid#237]
Is there any way to update cell in component level without updating parent state that used to draw grid. i noticed visual blinking issue on every cell update.

**Comment by alonshmiel** [comment]
Hi, can you please attach a codesandbox link with a minimal example?
Maybe I miss something but I don't see it's reproduced in the examples:
https://reactgrid.com/examples

**Comment by TarunGarapati** [comment]
@alonshmiel , @DLowHP , I think the question is if there is a UseState hook which contains rows and columns of a React Grid. And if there is any change to a cell, will it rerender the whole React Grid or just the updated cell.

@bayaroch-um please correct me if I'm wrong.

**Comment by DLowHP** [comment]
Updating the source data will tell React to rerender the whole React Grid, but in the process React does a lot of diffing under the hood to optimize the rendering process. If you would investigate the library codebase you'd find a lot of memoization techniques that work along the diffing algorithm to minimize updates.

TL;DR: It should only update the changed cell's DOM element. It's difficult to say what might've gone wrong without knowing the details.
